packages/create-sitecore-jss/src/initializers/nextjs/index.ts

```diff
@@ -54,6 +54,11 @@ export default class NextjsInitializer implements Initializer {
             name: 'nextjs-sxa - Includes example components and setup for Headless SXA projects',
             value: 'nextjs-sxa',
           },
+          {
```

Comment:
Wonder if these should actually be offered in a prompt from the [`nextjs-sxa` initializer](https://github.com/Sitecore/jss/blob/dev/packages/create-sitecore-jss/src/initializers/nextjs-sxa/index.ts#L19)? i.e. "Would you like to include any projects?" 

I think as long as the initializer returns the selected project add-ons on the `InitializerResults.initializers` prop, the init runner will handle it.